src/network/constants.py

```diff
@@ -0,0 +1,17 @@
+"""
+Network protocol constants
+"""
+
+
+# address is online if online less than this many seconds ago
+ADDRESS_ALIVE = 10800
```

Comment:
Maybe comment with `#:`?

Reply:
ok makes sense

Reply:
familiarising myself with sphinx ...

Reply:
Has this been resolved? I'm not sure I did it correctly.